README.md

```diff
@@ -97,9 +97,9 @@ ISO_IEC_13818-4_2004
     Description: ISO_IEC_13818-4_2004 ADTS conformance test suite
```

Comment:
I think we can include new test suites in this list... what do you think @mcesariniflu?

Reply:
- Related to PR in flu-plugins:
Ok. I will create a PR for flu-plugins modifying the names also.

- Related to ISO_IEC_13818-4_2004
I saw it but I didn't change it because it was not mentioned in this task.

- Related to the README.md file
I updated only the name of the test suit mentioned in this task because there is another task, which is to update the README.md file itself. See: https://fluendo.atlassian.net/browse/COM-12613
That's the reason why I just modified the necessary things to meet this task.

Reply:
Ok, perfect, I hadn't seen the task you commented or I didn't remember, honestly I think the ones I have commented about the test suites that are changing (ISO_IEC_13818-4_2004 -> MPEG2_AAC-ADIF) are related to this task, but I understand.

Regarding the flu-plugins task let me know if you need me to review

Reply:
Ok, I will update the ISO_IEC_13818-4_2004 to MPEG2_AAC-ADIF. It won't take too much effort.
Yes, I'll let you know so you can do the review.